<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HdbTest</class>
 <widget class="QWidget" name="HdbTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>862</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HdbTest</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="TPlotLightMarker" name="plot">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Configuration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="pbGetSources">
        <property name="text">
         <string>Get Archived Sources</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pbClearPlot">
        <property name="text">
         <string>Clear Plot</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="6">
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QTreeWidget" name="twArchived">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>5</horstretch>
           <verstretch>2</verstretch>
          </sizepolicy>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>sources</string>
          </property>
         </column>
        </widget>
        <widget class="QTreeWidget" name="twIntervals">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <column>
          <property name="text">
           <string>start date</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>stop date</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>archiver</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>per per mod</string>
          </property>
         </column>
        </widget>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>from:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateTimeEdit" name="dteStart">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>to:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QDateTimeEdit" name="dteStop">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pbClearCurve">
        <property name="text">
         <string>Clear Selected Curve</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="pbSetSource">
        <property name="text">
         <string>set source</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QCheckBox" name="cbLive">
        <property name="text">
         <string>Live</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
  <customwidget>
   <class>TPlotLightMarker</class>
   <extends>EPlotLightMarker</extends>
   <header>TPlotLightMarker</header>
  </customwidget>
  <customwidget>
   <class>EPlotLight</class>
   <extends>QwtPlot</extends>
   <header>EPlotLight</header>
  </customwidget>
  <customwidget>
   <class>EPlotLightMarker</class>
   <extends>EPlotLight</extends>
   <header>EPlotLightMarker</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
